### DISCUSSION TITLE
[Bug report]

### DISCUSSION THREAD

**Discussion by nalecu** [GetPublii/Publii#1877] [Bug Report] [answered]
### Operating system

Ubuntu 22.04

### Publii version

0.46.2

### Issue type

Something else

### Bug description

503 Service Unavailable when trying to download a premium theme that I have an active subscription for (specifically, Carla 2.1.0.0)

**Comment by bobmitro** [comment, marked as answer]
Write a ticket or contact with sales at getpublii dot com

**Comment by nalecu** [comment]
Done, thank you!
